src/vs/workbench/services/assignment/common/assignmentService.ts

```diff
@@ -51,7 +51,10 @@ class MementoKeyValueStorage implements IKeyValueStorage {
 	}
 }
 
-class WorkbenchAssignmentServiceTelemetry implements IExperimentationTelemetry {
+class WorkbenchAssignmentServiceTelemetry extends Disposable implements IExperimentationTelemetry {
+
+	private readonly _onDidUpdateAssignmentContext = this._register(new Emitter<void>());
```

Comment:
Btw we also have a `Relay` class you can use in cases like this

Reply:
Thanks for the hint, I've forgot about that one.